applications/nrf5340_audio/src/audio/audio_datapath.c

```diff
@@ -165,14 +167,19 @@ static size_t test_tone_size;
  *
  * @note	Used to adjust audio clock to account for drift.
  *
- * @param	sdu_ref_us	Timestamp for sdu.
+ * @param	sdu_ref_us	Timestamp for SDU.
  * @param	frame_start_ts	Timestamp for I2S.
  *
- * @return	err_us	Error in microseconds.
+ * @return	Error in microseconds (err_us).
  */
 static int32_t err_us_calculate(uint32_t sdu_ref_us, uint32_t frame_start_ts)
 {
 	bool err_neg = false;
+
+	if (IS_ENABLED(CONFIG_BT_LL_ACS_NRF53) && IS_ENABLED(CONFIG_TRANSPORT_BIS)) {
+		sdu_ref_us += CONFIG_AUDIO_FRAME_DURATION_US;
+	}
+
 	int64_t total_err = (sdu_ref_us - frame_start_ts);
```

Comment:
As discussed, might have to change this to an ``int``

Reply:
Cast uint32_t to (int64_t)

Reply:
Done